### ISSUE TITLE
APEQPT: why do we ask when it's being completed when we know? And consequences.

### ISSUE THREAD

**Issue by RudolfCardinal** [RudolfCardinal/camcops#219]
The APEQPT task (but not APEQ-CPFT-Perinatal) asks "Date & Time the Assessment Tool was Completed". This is a bit incongruous because

* creation datetime is automatically collected for all tasks, and this question is probably an accidental hangover from the paper version;
* the HTML server display doesn't show this field, ``q_datetime``;
* I noticed it because the task framework doesn't prepopulate ``q_datetime`` (unsurprisingly), but the FHIR framework relies on that field for its standard datetime field, rather than ``when_created``. So FHIR export crashes for this task in the unit tasks.

I'm working on the broader FHIR Questionnaire problem, but on this point -- should we just remove the question and field? It's something we ask the user that we already know automatically and it leads to some confusion. I don't think there is much real risk in deleting it from existing data (since it it effectively duplicates ``when_created``), but that's the point most worth considering. (There is a free-text comments field, so we could even do a database migration that appends a string containing this date/time to the free-text field before deleting the superfluous one... but that may be unnecessary.)

**Comment by martinburchell** [comment]
@RudolfCardinal To me it doesn't make sense to have this as a question unless the dates and times are likely to be different from `when_created`. I can imagine a scenario where on Wednesday a clinician might create an electronic record from a paper questionnaire filled in by the patient on Tuesday, but this wouldn't apply to an anonymous questionnaire on a tablet filled in by a patient.

The FHIR export could just use `when_created`, couldn't it?

**Comment by RudolfCardinal** [comment]
Yes, agree. I'll alter the FHIR bit now, but the APEQPT will be a separate database migration so I'll hang on for now on that.
